src/apolo_app_types/protocols/common/networking.py

```diff
@@ -39,3 +60,45 @@ class RestAPI(HttpApi):
 
 class GrpcAPI(HttpApi):
     api_type: Literal["grpc"] = "grpc"
+
+
+class OpenAICompatChatAPI(RestAPI):
```

Comment:
nit, we could move it to `openai_compat` module

Reply:
Sounds fair